src/CMakeLists.txt

```diff
@@ -175,18 +175,18 @@ ELSE(CRUX)
 
   # SET LIBRARIES FOR PERCOLATOR
   set(COMMON_LIBRARIES perclibrary ${BLAS_LIBRARIES} fido picked_protein ${XERCESC_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${CURL_LIBRARIES} ${RPC_LIBRARIES})
-  if(COVERAGE)
-    set(COMMON_LIBRARIES -fprofile-arcs ${COMMON_LIBRARIES})
-  endif(COVERAGE)
-  if(MSVC)
-    target_link_libraries(percolator ${COMMON_LIBRARIES})
-  else(MSVC)
-    if (APPLE)
-      target_link_libraries(percolator ${COMMON_LIBRARIES} ${OpenMP_CXX_LIBRARIES})
+  if(NOT MSVC)
+    if(APPLE)
+      set(COMMON_LIBRARIES ${COMMON_LIBRARIES} ${OpenMP_CXX_LIBRARIES})
     else(APPLE)
       target_link_libraries(percolator ${COMMON_LIBRARIES} stdc++)
```

Comment:
This is a bit confusing to me, should this line not be 
> set(COMMON_LIBRARIES ${COMMON_LIBRARIES} stdc++)

Reply:
Nice catch. Yes, this is an error introduced in a merge I think.